android/app/src/main/java/com/microsoft/codepush/common/exceptions/CodePushFinalizeException.java

```diff
@@ -7,11 +7,48 @@
  */
 public class CodePushFinalizeException extends IOException {
 
+    /**
+     * Type of the operation being performed before closing resources.
+     */
+    public enum OperationType {
+
+        DEFAULT("Error closing IO resources."),
+
+        COPY("Error closing IO resources when copying files. "),
```

Comment:
Please remove extra space character at the end of string here and below.

Reply:
Fixed.